Task/python/Taskfile.yaml

```diff
@@ -0,0 +1,53 @@
+# https://taskfile.dev
```

Comment:
Is it possible to do another level of includes here?  I wonder if a base `clean` is possible in `goat/Task/Taskfile.yaml`, which this uses and then adds on the python-specific items?

Reply:
I've added a base clean, currently it's leveraging python to clean because globstar isn't consistent across all shells. Will need to find a better way to deal with this but for now leveraging I think leveraging the repo python is okay.

Reply:
@seanctech I don't understand; if we can run a docker container then we control the shell and globstar config?